alacritty_terminal/src/term/mod.rs

```diff
@@ -58,6 +58,8 @@ struct RenderableCursor {
 pub struct CursorKey {
     pub style: CursorStyle,
     pub is_wide: bool,
+    /// For Blinking* cursors, true if the cursor is currently hidden
+    pub blinking_out: bool,
```

Comment:
This variable name is incredibly confusing, could we go with something more descriptive. A cursor being "out" is not something I would understand, so other people would probably have even more of a problem.

Reply:
It might be a bit of an indication that the way this is done isn't ideal.

I don't quite remember all our previous conversations, but have you tried storing the cursor visibility for blinking outside of alacritty_terminal? Currently I don't see why it's needed inside of it.